### PR TITLE
LPS-131090 Create TeamCTDisplayRenderer

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/LPS-131090

### CODE REVIEW

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+				if (Validator.isNotNull(userName)) {
+					return userName;
+				}
+
+				return null;
+			}
+		).display(
+			"description", team.getDescription()
+		).display(
+			"users",
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:109]: Let's call this "x-users".

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+			() -> _userLocalService.searchCount(
+				team.getCompanyId(), StringPool.BLANK,
+				WorkflowConstants.STATUS_ANY,
+				LinkedHashMapBuilder.<String, Object>put(
+					"usersTeams", team.getTeamId()
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:114]: We have a constants file for this one, see UserGroupFinderConstants.

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+					return userName;
+				}
+
+				return null;
+			}
+		).display(
+			"description", team.getDescription()
+		).display(
+			"users",
+			() -> _userLocalService.searchCount(
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:110]: Don't need "() ->"

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+				team.getCompanyId(), StringPool.BLANK,
+				WorkflowConstants.STATUS_ANY,
+				LinkedHashMapBuilder.<String, Object>put(
+					"usersTeams", team.getTeamId()
+				).build())
+		).display(
+			"user-groups",
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:117]: "x-user-groups"

Comment (Preston-Crary) [reply modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:117]: Will need to use Language.format for args.

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+	}
+
+	@Override
+	protected void buildDisplay(DisplayBuilder<Team> displayBuilder) {
+		Team team = displayBuilder.getModel();
+
+		displayBuilder.display(
+			"name", team.getName()
+		).display(
+			"created-by",
```
Comment (Preston-Crary) [inline modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:96]: Let's leave out "created-by" if it's not shown on the edit page.

File: modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java (hunk 4 above)
Comment (ctang3) [reply modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:117]: Just to clarify, you mean to have it display like this? This is what I pushed in my updated branch, but if you mean having the "x User Groups" in the left column and null for the right, let me know and I'll change it.  
image

Comment (Preston-Crary) [reply modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:109]: You're right, that would be weird. Let's leave that change out. In my mind it was only showing the x-users not the label on the left.

Comment (ctang3) [reply modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:109]: Updated, thanks!

Comment (ctang3) [reply modules/apps/site/site-teams-web/src/main/java/com/liferay/site/teams/web/internal/change/tracking/spi/display/TeamCTDisplayRenderer.java:107]: Updated, thanks!

**Comment by Preston-Crary** [comment]
ci:forward

**Comment by ctang3** [comment]
Fixed SF

**Comment by ctang3** [comment]
ci:forward

**Pull request closed without merging**